modules/core/02-client/keeper/keeper.go

```diff
@@ -459,8 +459,8 @@ func (k Keeper) ScheduleIBCSoftwareUpgrade(ctx sdk.Context, plan upgradetypes.Pl
 		return err
 	}
 
-	// emitting an event for handling client upgrade proposal
-	emitUpgradeClientProposalEvent(ctx, plan.Name, plan.Height)
+	// emitting an event for scheduling an upgrade plan
+	emitScheduleIBCSoftwareUpgradeEvent(ctx, plan.Name, plan.Height, cs)
```

Comment:
I'd be curious to know if hermes uses the existing event type. I forget if we checked on this?

Reply:
Maybe we will find out if the e2e tries to use hermes to complete the upgrade

Reply:
hermes just uses the `UpgradeClient` event, as i can see, not the `UpgradeClientProposal` event.

Reply:
Yea same here. I think they'd likely use `UpgradeChain` event instead as well